install/installing-knative-eventing.adoc

```diff
@@ -23,4 +23,8 @@ include::modules/serverless-install-kafka-odc.adoc[leveloffset=+1]
 [id="next-steps_installing-knative-eventing"]
 == Next steps
 
-* If you want to use Knative services you can xref:../install/installing-knative-serving.adoc#installing-knative-serving[install Knative Serving].
+* Learn more about how to use Knative event-driven architecture in xref:../eventing/knative-eventing.adoc#knative-eventing-overview[Chapter 4. Knative Eventing].
```

Comment:
I think "Chapter 4" could be dropped as I do not see any direct chapter references in the published documents.